doc/source/categorical.rst

```diff
@@ -44,11 +44,26 @@ The categorical data type is useful in the following cases:
 * As a signal to other Python libraries that this column should be treated as a categorical
   variable (e.g. to use suitable statistical methods or plot types).
 
+.. note::
+
+    In contrast to R's `factor` function, categorical data is not converting input values to
+    strings and categories will end up the same data type as the original values.
```

Comment:
Remove the "and" and split this in two sentences? Currently this could be read as two differences from R, while it's just 1 difference (not stringified), and the consequence (same dtype).

Reply:
Replaced the "and" with a semicolon instead of making separate sentences.